examples/cms-buttercms/.gitignore

```diff
@@ -32,3 +32,6 @@ yarn-error.log*
 
 # vercel
 .vercel
+
+# other
+.vscode
```

Comment:
The `.gitignore` file needs to remain the same as other examples

Reply:
this has been reverted